cart/cart.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/style.css" />
+    <link rel="stylesheet" href="/cart/cart.css" />
+    <title>Ecommmerce - Cart</title>
+  </head>
+  <body>
+    <section class="header-wrapper">
+      <header class="header">
+        <a href="$" class="header-brand h3">Some Business</a>
+        <nav class="header-nav">
+          <ul>
+            <li><a href="#">Categories</a></li>
+            <li><a href="#">Plans</a></li>
+          </ul>
+        </nav>
+        <div class="header-search-wrapper">
+          <label for="search"><i class="fas fa-search"></i></label>
+          <input type="text" id="search" />
+        </div>
+        <div class="header-ctas">
+          <button class="btn btn-outline-primary header-cart-btn">
+            <a href="#"><i class="fas fa-shopping-cart"></i></a>
+          </button>
+          <button class="btn btn-outline-primary">
+            <a href="#">Log in</a>
+          </button>
+          <button class="btn btn-solid-primary"><a href="#">Sign up</a></button>
+        </div>
+      </header>
+    </section>
+    <main>
+      <h1 class="cart-heading">Shopping Cart</h1>
+      <div class="cart-wrapper">
+        <section class="cart-bag">
+          <h1 class="h4 cart-wrapper-desc">2 courses in the cart.</h1>
+          <div class="hz-card-wrapper">
+            <div class="course-card flex">
```

Comment:
It will be difficult to review code if you define flexbox related properties partially here and partially in the separate stylesheet. Just put all together at one place.

Reply:
+1